main.R

```diff
@@ -7,9 +7,9 @@ library(readr)
 
 # input and output directories -------------------------------------------------
 
-input_dir <- "~/Desktop/test_pacta/test_outputs"
-output_dir <- "./output"
-data_dir <- "~/data/pactadatadev/workflow-data-preparation-outputs/2023Q4_20240701T114132Z"
+input_dir <- "./inputs"
+output_dir <- "./outputs"
+data_dir <- "./data"
```

Comment:
as usual, makes it harder to run w/o docker

Reply:
Given that the main intention is to spin this up on Azure and have it run a Docker image, I am ok with this

Reply:
If we're actually dockerizing this, we should package-ize it as well. I think it would be best to wrap the `main.R` script as a function that accepts these directories as inputs.

That's in line with what we've done with other workflow.* repos, and would allow both local and docker running.

Reply:
Happy to extract that idea as a new issue, but that's very much not a priority for me. 

Given that we are still in the "prototype"/"throwaway" phase with this work, I really just want this PR to be small and focused, that simply allows me to spin up a Docker image to run this script.